src/utils/DependsOnManager.ts

```diff
@@ -40,7 +39,7 @@ export class DependsOnManager {
     this.parentFacetRef = this.getParentFacet(this.facet.ref);
 
     if (this.parentFacetRef) {
-      this.facet.ref.bind.onQueryState(MODEL_EVENTS.CHANGE, undefined, () => this.resetIfConditionUnfullfiled());
+      $$(this.facet.ref.root).on('state:change', () => this.resetIfConditionUnfullfiled());
```

Comment:
Needed so that the condition execute even if the dependent facet is disabled.

The `.bind.onQueryState` is a special wrapper around the basic `.on`, which checks if the component is disabled before executing the callback.

Here, we want to possibly reset even if disabled.